### DISCUSSION TITLE
Use environmental variables inside directus

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17155] [Feature Requests]
Would be nice if we could use environmental variables similar to translation strings, so that we can insert them from a dropdown anywhere we need them.

Example (assuming to develop local and deploy on a server):

I want to create a preview link for my items, to view the content on the front-end. For the local environment the Preview URL will be something like `http://localhost:3000/items/{{item-slug}}?preview` and for the remote env it will be something like `https://www.example.com/items/{{item-slug}}?preview`. Wouldn’t it be handy to use something like `{{project-url}}/items/{{item-slug}}?preview` and set the `project-url` via environmental variables (e.g. .env file)?

**Comment by ghost** [comment]
Maybe an environmental variable for `PROJECT_URL` would be enough?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Reply by haidv1992** [reply]
directus like wordpress  all things save in database . When deploy  not smart to migration
